libs/coin-modules/coin-polkadot/src/network/node/nominations.ts

```diff
@@ -24,37 +23,44 @@ export const fetchNominations = async (address: string): Promise<SidecarNominati
       targets: [],
     };
   }
-
   const { targets, submittedIn } = nominationsOpt.unwrap();
 
-  const [exposures, stashes] = await Promise.all([
-    api.query.staking.erasStakers.multi<Exposure>(targets.map(target => [activeEra, target])),
-    api.derive.staking.stashes(),
-  ]);
+  const stashes = await api.derive.staking.stashes();
 
   const allStashes = stashes.map(stash => stash.toString());
+  let returnTargets = [];
+  for (const target of targets) {
+    let status: "active" | "inactive" | "waiting" | null = null;
+    let value = "0";
+    const exposure = await api.query.staking.erasStakersOverview(activeEra, target.toString());
+    if (exposure.isNone) {
+      if (allStashes.includes(target.toString())) {
+        status = "waiting";
+      }
+    } else {
+      const pageCount:number = (exposure.toJSON() as any).pageCount ?? 0;
+      for (let i = 0; i < pageCount; i++) {
+        const nominators = (await api.query.staking.erasStakersPaged(activeEra, target.toString(), i)).unwrap().others;
```

Comment:
just curious, the documentation on `erasStakersPaged` (https://polkadot.js.org/docs/substrate/storage/#erasstakerspagedu32-accountid32-u32-optionspstakingexposurepage) mentions that "it should only be accessed through EraInfo.". But the documentation is light on what EraInfo is and where we could get it, do you have any idea?

Reply:
In my opinion, "EraInfo" is the "ActiveEraInfo" which is defined here:
https://github.com/polkadot-js/api/blob/dac94c51964a90f9b26bc88d5a63f1e1b2038281/packages/types/src/interfaces/staking/types.ts#L9

"only be accessed through EraInfo" refer to the first parameter in "api.query.staking.erasStakersPaged"

we get this index of current Era with this:
https://polkadot.js.org/docs/substrate/storage/#activeera-optionpalletstakingactiveerainfo
It's what we have done in line 9
```
api.query.staking.activeEra.at(hash),
```